src/multi.c

```diff
@@ -55,6 +55,13 @@ void freeClientMultiState(client *c) {
     zfree(c->mstate.commands);
 }
 
+void resetClientMultiState(client *c) {
```

Comment:
Nit: We can use `static` to mark that it's internal to this file.

```suggestion
static void resetClientMultiState(client *c) {
```

Reply:
Just like free and init, maybe reset will be called somewhere else someday, so I won't add it to avoid adding and removing back and forth. Haha.